source/_components/image_processing.facebox_face_detect.markdown

```diff
@@ -0,0 +1,61 @@
+---
+layout: page
+title: "Facebox"
+description: "Detect and recognise faces with Facebox."
+date: 2018-05-03 00:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: machine-box-alpha.png
+ha_category: Image Processing
+featured: false
+ha_release: 0.70
+---
+
+The `facebox` image processing component allows you to detect and recognise faces in a camera image using [Facebox](https://machinebox.io/docs/facebox). The state of the entity is the number of faces detected, and recognised faces are listed in the `matched_faces` attribute. Facebox runs in a Docker container, and it is recommended that you run this container on a machine with a minimum of 2 GB RAM. To minimise the time for Facebox to detect faces in an image using this component, on your machine with Docker run Facebox with:
+```
+MB_KEY="INSERT-YOUR-KEY-HERE"
+
+sudo docker run --name=facebox --restart=always 8080:8080 -e "MB_KEY=$MB_KEY"  machinebox/facebox
+```
+
+To enable this component in your installation, add the following to your `configuration.yaml` file:
+
+{% raw %}
```

Comment:
Do we need the raw tags? I don't believe so.

Reply:
ok